src/components/table.js

```diff
@@ -98,6 +99,46 @@ const paginator = function(props) {
   ])
 }
 
+const defaultComponents = {
+  header: {
+    row: props => tr(_.merge({
+      style: {
+        border: '1px solid #CCCCCC',
+        backgroundColor: '#FAFAFA'
+      }
+    }, props)),
+    cell: props => td({
+      style: {
+        alignment: 'left',
```

Comment:
Is `alignment` a real thing?

Reply:
Nope. Coulda sworn it was doing something at some point, but it's not any more.